src/components/PaymentDetailsDialog.ts

```diff
@@ -22,52 +22,60 @@ export class PaymentDetails {
     this.successSnackbar = page.locator('div.snackbar.success');
   }
 
-  async submit(): Promise<void> {
-    await test.step(`Clicks on "Submit" button`, async() => {
+  async clickSubmitButton(): Promise<void> {
+    await test.step(`Click on "Submit" button`, async() => {
       await this.submitButton.click();
     });
   }
 
-  async close(): Promise<void> {
-    await test.step(`Closes the dialog`, async() => {
+  async clickCloseButton(): Promise<void> {
+    await test.step(`Close the dialog`, async() => {
```

Comment:
Ideally step should describe same actions as method does, for ex - "Click Close dialog button"

Reply:
fixed